app/src/main/java/com/github/braillesystems/learnbraille/data/dsl/Courses.kt

```diff
@@ -1,21 +1,23 @@
 package com.github.braillesystems.learnbraille.data.dsl
 
 import com.github.braillesystems.learnbraille.data.entities.Course
+import com.github.braillesystems.learnbraille.data.entities.CourseDesc
+import com.github.braillesystems.learnbraille.data.entities.CourseName
+import com.github.braillesystems.learnbraille.data.entities.DBid
 import com.github.braillesystems.learnbraille.utils.side
 
-sealed class CourseID(val id: Long)
+sealed class CourseID(val id: DBid)
```

Comment:
I've got used to using `typealias` for shortening long types, e. g. `typealias Giraffe=Tuple<Head, Tail, Body, List<Leg>>`.  See [Kotlin docs](https://kotlinlang.org/docs/reference/type-aliases.html)<br>
Using typealiases for "primitive" types, to my mind, makes code unnecessarily intricate, and maybe it would have been better to write `val DBid: Long` than `val id: DBid`.

Reply:
https://www.reddit.com/r/Kotlin/comments/5dzi6m/question_when_should_typealias_be_used_and_when/da9f5uh/?utm_source=reddit&utm_medium=web2x&context=3

Reply:
Yes, this thing is controversial a bit. But I think that domain-specific type aliases improve readability a lot and make it possible to abstract domain type over its particular implementation.